### PR TITLE
add pibakery (new formula)

### PR DESCRIPTION
**Pull request by kai5263499**
### Checklist
- [X] The commit message includes the cask’s name and version.
- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:
- [X] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [X] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [X] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/pibakery.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'pibakery' do
+  version '0.2.3'
+  sha256 '655c01afafdd3012332ad3a68f0388f5f838cf826dc49b7d645b5b0928e01f50'
+
+  # github.com/davidferguson was verified as official when first introduced to the cask
+  url 'https://github.com/davidferguson/pibakery/releases/download/v0.2.3/PiBakery.pkg'
```
Comment (reitermarkus) [inline Casks/pibakery.rb]: This needs an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/pibakery.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  name 'PiBakery'
+  homepage 'http://www.pibakery.org/'
+  license :gpl
+
+  pkg 'PiBakery.pkg'
```
Comment (reitermarkus) [inline Casks/pibakery.rb]: `pkg` installers should have a corresponding [`uninstall pkgutil`](https://github.com/caskroom/homebrew-cask/blob/15171687a14436e6bde53c3a1111c8602f30e9ae/doc/cask_language_reference/stanzas/uninstall.md#uninstall-pkgutil-is-the-easiest-and-most-useful).

**Review by reitermarkus — CHANGES_REQUESTED** [review]

**Comment by kai5263499** [comment]
I've completed both requests

File: Casks/pibakery.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+cask 'pibakery' do
+  version '0.2.3'
+  sha256 '655c01afafdd3012332ad3a68f0388f5f838cf826dc49b7d645b5b0928e01f50'
+
+  # github.com/davidferguson was verified as official when first introduced to the cask
+  url 'https://github.com/davidferguson/pibakery/releases/download/v0.2.3/PiBakery.pkg'
```
Comment (fanquake) [inline Casks/pibakery.rb]: You can also use #{version} in the URL stanza

**Comment by kai5263499** [comment]
I've updated the PR to include #{version} in the url stanza

**Review by adidalal — APPROVED** [review]

**Pull request merged**